doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -835,6 +840,38 @@ This update policy applies to all listed stop trigger types.
 
+.. _ug_bt_fast_pair_gatt_service_fmdn_dult_motion_detector:
+
+Interacting with the motion detector from DULT
+----------------------------------------------
+
+The motion detector is an optional feature of the DULT subsystem that can be integrated into the FMDN extension.
+For more details about the feature, see the `DULT Motion detector`_ section of the DULT specification.
+You can enable the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` Kconfig option to support the DULT motion detector functionality in the FMDN extension.
+The FMDN extension implementation acts as a thin wrapper for the DULT motion detector module callbacks.
+It passes callbacks from the DULT motion detector module to the user application.
+
+To register the motion detector callbacks, use the :c:func:`bt_fast_pair_fmdn_motion_detector_cb_register` function.
+All motion detector callbacks defined in the :c:struct:`bt_fast_pair_fmdn_motion_detector_cb` structure are mandatory to register:
+
+* The motion detector start request is indicated by the :c:member:`bt_fast_pair_fmdn_motion_detector_cb.start` callback.
+  After this callback is called, the motion detector events are polled periodically with the :c:member:`bt_fast_pair_fmdn_motion_detector_cb.period_expired` callback.
+  Typical action after the motion detector start request is to power up the accelerometer and start collecting motion data.
+* The motion detector period expired event is indicated by the :c:member:`bt_fast_pair_fmdn_motion_detector_cb.period_expired` callback.
+  This callback is called at the end of each motion detector period.
+  The :c:member:`bt_fast_pair_fmdn_motion_detector_cb.start` callback indicates the beginning of the first motion detector period.
+  The next period is started as soon as the previous period expires.
+  The user should notify the DULT module if motion was detected in the previous period.
+  The return value of this callback is used to pass this information.
+  The motion must be considered as detected if it fulfills the requirements defined in the `DULT Motion detector`_ section of the DULT documentation.
+* The motion detector stop request is indicated by the :c:member:`bt_fast_pair_fmdn_motion_detector_cb.stop` callback.
+  It concludes the motion detector activity that was started by the :c:member:`bt_fast_pair_fmdn_motion_detector_cb.start` callback.
+  Typical action after the motion detector stop request is to power down the accelerometer.
+
+The motion detector is started by the DULT subsystem when the accessory is in the separated state for a sufficient amount of time.
+When the motion is detected during the motion detector active period, the :c:member:`bt_fast_pair_fmdn_ring_cb.start_request` callback is called to request the ringing action.
+It is done to inform the non-owner that is moving the accessory, that the device might have been used to track the non-owner.
```

Comment:
You can also use the sentence from the DULT integration guide. Perhaps it will be clearer

Reply:
Your sentence sounds quite good.